arch/arm64/boot/dts/freescale/imx8mp-leica-beluga-0-5.dts

```diff
@@ -0,0 +1,556 @@
+			};
+
+			BUCK4 {
+				regulator-name = "BUCK4";
+				regulator-min-microvolt = <3000000>;
+				regulator-max-microvolt = <3600000>;
+				regulator-boot-on;
+				regulator-always-on;
+			};
+
+			BUCK5 {
+				regulator-name = "BUCK5";
+				regulator-min-microvolt = <1650000>;
+				regulator-max-microvolt = <1950000>;
+				regulator-boot-on;
+				regulator-always-on;
+			};
+
+			BUCK6 {
+				regulator-name = "BUCK6";
+				regulator-min-microvolt = <1045000>;
+				regulator-max-microvolt = <1155000>;
+				regulator-boot-on;
+				regulator-always-on;
+			};
+
+			LDO1 {
+				regulator-name = "LDO1";
+				regulator-min-microvolt = <1650000>;
+				regulator-max-microvolt = <1950000>;
+				regulator-boot-on;
+				regulator-always-on;
+			};
+
+			LDO3 {
+				regulator-name = "LDO3";
+				regulator-min-microvolt = <1710000>;
+				regulator-max-microvolt = <1890000>;
+				regulator-boot-on;
+				regulator-always-on;
+			};
+
+			LDO5 {
+				regulator-name = "LDO5";
+				regulator-min-microvolt = <1800000>;
+				regulator-max-microvolt = <3300000>;
+				regulator-boot-on;
+				regulator-always-on;
+			};
+		};
+	};
+
+	rtc@32 {
+		compatible = "epson,rx8130";
+		reg = <0x32>;
+		aux-voltage-chargeable = <0>;
+		wakeup-source;
+	};
+
+	pcieclk: clk@68 {
+		compatible = "renesas,9fgv0241";
+		reg = <0x68>;
+		clocks = <&clk_xtal25>;
+		#clock-cells = <1>;
+	};
+};
+
+&i2c2 {
+	clock-frequency = <400000>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_i2c2>;
+	status = "okay";
+
+	at24c512: eeprom@57 {
+		compatible = "atmel,24c512";
+		reg = <0x57>;
+		pagesize = <16>;
+	};
+
+	tca6424a: gpio-controller@22 {
+		compatible = "ti,tca6424";
+		reg = <0x22>;
+		gpio-controller;
+		#gpio-cells = <2>;
+		interrupt-parent = <&gpio3>;
+		interrupts = <26 IRQ_TYPE_LEVEL_LOW>;
+		pinctrl-names = "default";
+		pinctrl-0 = <&pinctrl_io_expdr>;
+		interrupt-controller;
+		reset-gpios = <&gpio4 4 GPIO_ACTIVE_LOW>;
+		gpio-line-names = "", "", "",
+			"TD_GPIO",
+			"IMX_VBUS_EN","", "",
+			"IMX_TILT_PWR_EN",
+
+			"KDU_POWER_ONOFF",
+			"IMX_KDU1_LVDS_EN",
+			"IMX_KDU2_LVDS_EN",
+			"IMX_KDI_xRST",
+			"TD_PRESENT",
+			"IMX_EXTT_EN",
+			"IMX_FPGA_SW_nRST",
+			"IMX_FPGA_SPARE_GPIO_DIR",
+
+			"IMX_WLESS_EN",
+			"IMX_WLESS_nPD",
+			"IMX_WLESS_WL_xRST",
+			"IMX_WLESS_BT_xRST",
+			"IMX_TSR_xRST",
+			"IMX_AUD_xINT",
+			"IMX_AUD_xRST",	"";
+	};
+};
+
+&i2c4 {
+	clock-frequency = <400000>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_i2c4>;
+	status = "okay";
+};
+
+&i2c5 {
+	clock-frequency = <400000>;
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_i2c5>;
+	status = "okay";
+};
+
+&i2c6 {
+	pinctrl-names = "default";
+	pinctrl-0 = <&pinctrl_i2c6>;
+	clock-frequency = <400000>;
+	status = "okay";
+
+	tmp116@48 {
+		compatible = "ti,tmp116";
+		reg = <0x48>;
+		label = "ProcessorBoard:1";
+	};
+
+	tmp116@49 {
+		compatible = "ti,tmp116";
+		reg = <0x49>;
+		label = "ProcessorBoard:2";
+	};
+
+	tmp116@4A {
+		vcc-supply = <&reg_ext_vdd_3v3>;
```

Comment:
vcc-supply is not supported by the driver right now, also first two properties should be "compatible" followed by "reg" like you did above.